Userland/Libraries/LibWeb/DOM/StyleElementUtils.cpp

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright (c) 2023, Preston Taylor <95388976+PrestonLTaylor@users.noreply.github.com>
```

Comment:
You can keep it of course, but I do find it funny having a noreply email here :laughing: